frappe/__init__.py

```diff
@@ -129,7 +129,7 @@ def as_unicode(text: str, encoding: str = "utf-8") -> str:
 	elif isinstance(text, bytes):
 		return str(text, encoding)
 	else:
-		return str(text)
```

Comment:
This is definitely not equivalent. Few more I spotted too. Let's run rule by rule if possible with separate commits.

Reply:
I am guessing it's using incorrect type annotations to reach to conclusion that it's not required.